Helm integration test is failing on pre-k8s-1.16 due to invalid schema

<!-- **Are you in the right place?**
1. For issues or feature requests, please create an issue in this repository.
2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://slack.rook.io/).
3. Did you already search the existing open issues for anything similar? -->

**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
The helm integration test has been failing since [build 2395](https://jenkins.rook.io/blue/organizations/jenkins/rook%2Frook/detail/master/2395/pipeline/). As seen in that [test log](https://jenkins.rook.io/blue/rest/organizations/jenkins/pipelines/rook/pipelines/rook/branches/master/runs/2395/nodes/61/steps/114/log/?start=0):
```
2020-11-18 11:49:51.965350 I | testutil: Running kubectl [create -f -]
The CephCluster "test-cluster" is invalid: []: Invalid value: map[string]interface {}{"apiVersion":"ceph.rook.io/v1", "kind":"CephCluster", "metadata":map[string]interface {}{"creationTimestamp":"2020-11-18T11:49:52Z", "generation":1, "name":"test-cluster", "namespace":"helm-ns", "uid":"c5eb4bdc-f11c-46dd-92ff-71ac1182fdbc"}, "spec":map[string]interface {}{"cephVersion":map[string]interface {}{"allowUnsupported":false, "image":"ceph/ceph:v14.2.12"}, "dashboard":map[string]interface {}{"enabled":true}, "dataDirHostPath":"/home/ubuntu/workspace/rook_rook_master/rook-test/helm-ns/test-129268764", "healthCheck":map[string]interface {}{"daemonHealth":map[string]interface {}{"mon":map[string]interface {}{"interval":"10s", "timeout":"15s"}, "osd":map[string]interface {}{"interval":"10s"}, "status":map[string]interface {}{"interval":"5s"}}}, "metadataDevice":interface {}(nil), "mgr":map[string]interface {}{"modules":[]interface {}{map[string]interface {}{"enabled":true, "name":"pg_autoscaler"}, map[string]interface {}{"enabled":true, "name":"rook"}}}, "mon":map[string]interface {}{"allowMultiplePerNode":true, "count":1}, "network":map[string]interface {}{"hostNetwork":false}, "skipUpgradeChecks":true, "storage":map[string]interface {}{"config":map[string]interface {}{"databaseSizeMB":"1024", "journalSizeMB":"1024", "storeType":"bluestore"}, "deviceFilter":interface {}(nil), "useAllDevices":true, "useAllNodes":true}}}: validation failure list:
spec.storage.deviceFilter in body must be of type string: "null"

...

--- FAIL: TestCephHelmSuite (123.84s)
    ceph_installer.go:511: 
        	Error Trace:	ceph_installer.go:511
        	            				ceph_installer.go:495
        	            				ceph_base_deploy_test.go:188
        	            				ceph_base_deploy_test.go:178
        	            				ceph_base_deploy_test.go:161
        	            				ceph_helm_test.go:80
        	            				suite.go:102
        	            				ceph_helm_test.go:52
        	Error:      	Received unexpected error:
        	            	failed to get ceph cluster. cephclusters.ceph.rook.io "test-cluster" not found
        	Test:       	TestCephHelmSuite
    ceph_installer.go:541: 
        	Error Trace:	ceph_installer.go:541
        	            				ceph_installer.go:495
        	            				ceph_base_deploy_test.go:188
        	            				ceph_base_deploy_test.go:178
        	            				ceph_base_deploy_test.go:161
        	            				ceph_helm_test.go:80
        	            				suite.go:102
        	            				ceph_helm_test.go:52
        	Error:      	Received unexpected error:
        	            	failed to wait for clean up jobs to complete. timed out waiting for the condition
        	Test:       	TestCephHelmSuite
```

This looks related to the test method `getDeviceFilter()` in #6506. It is failing when a null value is not supported in the v1beta1.

This appears to only be a test issue.

**Expected behavior:**
Integration tests should pass on all k8s versions

**How to reproduce it (minimal and precise):**
<!-- Please let us know any circumstances for reproduction of your bug. -->

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary
* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

 To get logs, use `kubectl -n <namespace> logs <pod name>`
When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
Read [Github documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Storage backend version (e.g. for ceph do `ceph -v`):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/rook/master/ceph-toolbox.html)):

--- a/tests/framework/installer/environment.go
+++ b/tests/framework/installer/environment.go
@@ -71,7 +71,7 @@ func TestScratchDevice() string {
 
 // getDeviceFilter get the device name used for OSD
 func getDeviceFilter() string {
-	return getEnvVarWithDefault("DEVICE_FILTER", "")
+	return getEnvVarWithDefault("DEVICE_FILTER", `""`)
 }
 
 func getEnvVarWithDefault(env, defaultValue string) string {

